dotnet/test/DotNetExtensions/AzureFunctions/test/client.exe.config

```diff
@@ -0,0 +1,101 @@
+<configuration>
+   <configSections>
+      <sectionGroup  name="datastores">
+         <section  name="Default" type="System.Configuration.NameValueSectionHandler" requirePermission="false"/>
+      </sectionGroup>
+      <sectionGroup  name="languages" type="System.Configuration.NameValueSectionHandler">
+         <section  name="English" type="System.Configuration.SingleTagSectionHandler" requirePermission="false"/>
+      </sectionGroup>
+      <section  name="log4net" type="log4net.Config.Log4NetConfigurationSectionHandler, log4net" requirePermission="false"/>
+   </configSections>
+   <datastores>
+      <add  key="Connection-Default-DBMS" value="sqlserver"/>
+      <add  key="Connection-Default-Port" value=""/>
+      <add  key="Connection-Default-LockTimeout" value="0"/>
+      <add  key="Connection-Default-LockRetryCount" value="10"/>
+      <add  key="Connection-Default-IsolationLevel" value="CR"/>
+      <add  key="Connection-Default-Datasource" value="LcSFozUYYEwCMNTWIBzPxzBLm+G9NbFezqdOu8D8G4z="/>
+      <add  key="Connection-Default-User" value="L3KpX01Y+7yriRShRiH2vS=="/>
+      <add  key="Connection-Default-Password" value="L3KpX01Y+7yriRShRiH2vS=="/>
```

Comment:
Are these valid when used outside the tests?

Reply:
They aren't. Removed encrypted values.